README.md

```diff
@@ -15,10 +15,10 @@
 
 ![Added](images/multipleplatform.gif)
 
-## What's New (v1.10.0)
+## What's New (v1.11.0)
 ### Added
-* Add sample gallery page. User can view and create Azure IoT Edge solution based on samples.
-* Add issue template
+* Add IoT Edge marketplace. User can view and create Edge modules from marketplace.
```

Comment:
Please make sure the wording here is consistent with the ones in changelog after modification. Thank you.